BGL/test/BGL/test_Face_filtered_graph.cpp

```diff
@@ -278,7 +278,7 @@ void test_index_property_maps(const Graph& g)
 
   typedef typename boost::graph_traits<Graph>::face_descriptor g_face_descriptor;
   std::map<g_face_descriptor, std::size_t> map;
-  PMP::connected_components(g, boost::make_assoc_property_map(map), CGAL::parameters::all_default());
+  PMP::connected_components(g, boost::make_assoc_property_map(map), CGAL::parameters::use_default_values());
```

Comment:
[`CGAL::parameters::use_default_values()` is documented](https://doc.cgal.org/latest/BGL/group__bgl__namedparameters.html). I think you should not replace it everywhere, to keep it tested.

Reply:
I added a test.